content/integrations/aws.md

```diff
@@ -67,9 +55,9 @@ There are a number of other AWS services that are also available in Datadog but
 
 # Installation
 
-There are two integration methods that can be used to allow Datadog to monitor your AWS environment. Both require creating a policy in the AWS Console with a certain set of permissions. The difference between the two is whether you choose to create a role that Datadog has access to which is preferred due to the higher level of security, or create a user and share an AWS Secret and Access Key. To get a better understanding of role delegation, refer to the [AWS IAM Best Practices guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles).
+Setting up the Datadog integration with Amazon Web Services requires configuring role delegation using AWS IAM. To get a better understanding of role delegation, refer to the [AWS IAM Best Practices guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles).
 
-Note: GovCloud does not support role based authentication.
+Note: If you are using GovCloud or have resources on AWS China, contact support to help set up your integration.
```

Comment:
Note: The GovCloud and China reasons do not currently support IAM role delegation.  If you are deploying in these reasons please contact support to help set up your AWS integration via API keys.